extensions/amp-geo/0.1/amp-geo.js

```diff
@@ -224,18 +224,18 @@ export class AmpGeo extends AMP.BaseElement {
    * @return {boolean}
    */
   checkGroup_(countryGroup) {
-    let presetCountries = [];
-    countryGroup = countryGroup.map((country, idx) => {
+    countryGroup = countryGroup.reduce((countries, country) => {
+      // If it's a valid preset then we expand it.
       if (/^preset-/.test(country)) {
         user().assert(
             isArray(ampGeoPresets[country]),
-            `<${TAG}> preset ${country} not found`);
-        delete countryGroup[idx];
-        presetCountries = presetCountries.concat(ampGeoPresets[country]);
-        return;
+            '<amp-geo> preset ${country} not found');
```

Comment:
Oh, and `${country}` won't expand here, it's the literal string `'${country}'`.

Reply:
doh! fixed along with your other comments.